projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.ts

```diff
@@ -46,6 +47,7 @@ export class ShippingAddressComponent implements OnInit {
     this.isLoading$ = this.userAddressService.getAddressesLoading();
     this.existingAddresses$ = this.userAddressService.getAddresses();
     this.selectedAddress$ = this.checkoutDeliveryService.getDeliveryAddress();
+    this.changeSelectedAddressLoaderState$ = this.checkoutDeliveryService.getLoadSupportedDeliveryModeProcess();
```

Comment:
Is `getLoadSupportedDeliveryModeProcess` the right process to observe for setting the delivery address?  
In case not, There is a `getSetDeliveryAddressProcess` that, from the name, seems to monitor the selection of a delivery address.